src/core/server/saved_objects/import/utils.ts

```diff
@@ -9,12 +9,14 @@ import { SavedObject, SavedObjectsClientContract } from '../types';
 export interface UpdateDataSourceNameInVegaSpecProps {
   spec: string;
   newDataSourceName: string;
+  spacing?: number;
 }
 
 export const updateDataSourceNameInVegaSpec = (
   props: UpdateDataSourceNameInVegaSpecProps
 ): string => {
-  const { spec } = props;
+  const { spec, spacing } = props;
+  const stringifiedSpacing = spacing || 2;
```

Comment:
`stringifiedSpacing` is how much indenting should occur when an HJSON document is stringified. It's set to `2` as that seems to be the default for most Vega visualizations. If it were `0` there would be no indenting between the lines so it would look something like this:

```
{
data: {
url: {
index: some_index
body: {
...
}
}
}
...
}
```

Reply:
Can we add those explanation as comments?